socs/agents/scpi_psu/agent.py

```diff
@@ -5,18 +5,21 @@
 from ocs import ocs_agent, site_config
 from ocs.ocs_twisted import TimeoutLock
 
-from socs.agents.scpi_psu.drivers import PsuInterface
+from socs.agents.scpi_psu.drivers import PsuInterface, ScpiPsuInterface
 
 
 class ScpiPsuAgent:
-    def __init__(self, agent, ip_address, gpib_slot):
+    def __init__(self, agent, ip_address, gpib_slot, **kwargs):
```

Comment:
Sort of along the same lines as the below comment about the `port` arg, there's the question of how to handle the `gpib_slot` arg. Since one or the other is required, not both. Perhaps we can also make `gpib_slot` optional, but throw an error if neither is submitted. This avoids having an agent instance that's going to use the new Ethernet only interface required to have a `--gpib-slot` argument to function.